README.md

```diff
@@ -1,27 +1,274 @@
+target, and configuration files are hosted on the Hubverse's Amazon Web Services (AWS) infrastructure,
+in a public S3 bucket:
+*[insert S3 bucket name, as specified in admin.json cloud.host.storage_location]*
+
+**Note**: For efficient storage, all model-output files in S3 are stored in parquet format, even if the original
+versions in the GitHub repository are .csv.
+
+GitHub remains the primary interface for operating the hub and collecting forecasts from modelers.
+However, the mirrors of hub files on S3 are the most convenient way to access hub data without using git/GitHub or
+cloning the entire hub to your local machine.
+
+The sections below provide examples for accessing hub data on the cloud, depending on your goals and
+preferred tools. The options include:
+
+| Access Method              | Description                                                                           |
+| -------------------------- | ------------------------------------------------------------------------------------- |
+| hubData (R)                | Hubverse R client and R code for accessing hub data                                   |
+| Polars (Python)            | Python open-source library for data manipulation                                      |
+| AWS command line interface | Download hub data to your machine and use hubData or Polars for local access          |
+
+In general, accessing the data directly from S3 (instead of downloading it first) is more convenient. However, if
+performance is critical (for example, you're building an interactive visualization), or if you need to work offline,
+we recommend downloading the data first.
+
+<!-------------------------------------------------- hubData ------------------------------------------------------->
+
+<details>
+
+<summary>hubData (R)</summary>
+
+[hubData](https://hubverse-org.github.io/hubData), the Hubverse R client, can create an interactive session
+for accessing, filtering, and transforming hub model output data stored in S3.
+
+hubData is a good choice if you:
+
+- already use R for data analysis
+- want to interactively explore hub data from the cloud without downloading it
+- want to save a subset of the hub's data (*e.g.*, forecasts for a specific date or target) to your local machine
+- want to save hub data in a different file format (*e.g.*, parquet to .csv)
+
+### Installing hubData
+
+To install hubData, follow the [instructions in its documentation](https://hubverse-org.github.io/hubData/#installation).
+
+### Using hubData
+
+hubData's [`connect_hub` function](https://hubverse-org.github.io/hubData/reference/connect_hub.html) returns an Arrow
+[multi-file dataset](https://arrow.apache.org/docs/r/reference/Dataset.html) that represents a hub's model output data.
+The dataset can be filtered and transformed using dplyr and then materialized into a local data frame
+using [`collect_hub`](https://hubverse-org.github.io/hubData/reference/collect_hub.html).

```

Comment:
1. give the functions their parens (https://style.tidyverse.org/news.html#code-style)
2. give the multi-file data set its context

```suggestion
hubData's [`connect_hub()` function](https://hubverse-org.github.io/hubData/reference/connect_hub.html) returns an [Arrow
multi-file dataset](https://arrow.apache.org/docs/r/reference/Dataset.html) that represents a hub's model output data.
The dataset can be filtered and transformed using dplyr and then materialized into a local data frame
using the [`collect_hub()` function](https://hubverse-org.github.io/hubData/reference/collect_hub.html).

```